.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report a bug in this library
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+This form is for bug reports for this library only.
+For general, language-agnostic questions and troubleshooting, please ask/look for answers [here](https://groups.google.com/g/adwords-api).
+
+Please check existing issues to see if your bug has already been reported before creating a new issue. We encourage you to add any relevant information to the existing issue if one already exists.
+-->
+
+**Which version of the client library are you using?**
+
+**Which version of PHP are you using?**
+
+**Which operating system (Linux, Windows, ...) and version?**
```

Comment:
@devchas what about adding a recommended way of getting these pieces of information? This would help to get accurate, complete and nicely formatted information.

Reply:
For now, I think we should leave as is for the sake of brevity. If we find this becomes problematic when receiving responses, let's reconsider.

Reply:
I agree it is not a blocker so will mark it as resolved but I also think that it would be useful so would love us to follow up on this later.